src/domain/event/utils.ts

```diff
@@ -1101,10 +1101,13 @@ export const checkCanUserDoAction = ({
     case EVENT_ACTIONS.ACCEPT_AND_PUBLISH:
     case EVENT_ACTIONS.PUBLISH:
     case EVENT_ACTIONS.UPDATE_PUBLIC:
+    case EVENT_ACTIONS.SEND_EMAIL:
       return isAdminUser;
     case EVENT_ACTIONS.UPDATE_DRAFT:
       return isRegularUser || isAdminUser;
   }
+
+  return false;
```

Comment:
This return appears to be unreachable, probably should be moved to default case of the switch statement 🤔